kits/daisy/getJoyCommands.m

```diff
@@ -1,46 +1,44 @@
-function [xyzVel, rotVel, auxCmd] = getJoyCommands( joy )
+function [xyzVel, rotVel, auxCmd] = getJoyCommands( fbkIO )
 
-    % Assumes 'DUALSHOCK�4 USB Wireless Adapto'
+    % Assumes you're using the HEBI Mobile I/O App
 
     xyzVel = zeros(3,1);
     rotVel = zeros(3,1);
     auxCmd = struct();
 
-    [axes, buttons, povs] = read(joy);
+    QUIT_BUTTON = 'b8';
+    STANCE_MODE = 'b1';
 
-    QUIT_BUTTON = 11;
-    STEP_TOGGLE = 12;
-
-    % PRESS DOWN LEFT JOYSTICK TO QUIT
-    if buttons(QUIT_BUTTON)
+    % PRESS DOWN B8 TO QUIT
+    if fbkIO.(QUIT_BUTTON)
         auxCmd.quit = true;
     else
         auxCmd.quit = false;
     end
     
     % PRESS A TO TOGGLE STEPPING ON AND OFF
-    if buttons(STEP_TOGGLE)
-        auxCmd.toggleStepping = true;
+    if fbkIO.(STANCE_MODE)
+        auxCmd.steppingMode = false;    
     else
-        auxCmd.toggleStepping = false;
+        auxCmd.steppingMode = true;
     end
     
     joyXYZScale = .3;
     joyRotScale = .6;
-    joyAxisDeadZone = .06;
+    joyAxisDeadZone = .05;
```

Comment:
I don't think the deadzone is really necessary with the Mobile IO approach, unless you are trying to minimize cross-talk between axes when trying to move in a straight line.